kpatch-build/args.h

```diff
@@ -0,0 +1,41 @@
+#include <argp.h>
+
+extern const int NUM_ARGS;
+
+struct arguments {
+	int debug;
+	char **args;
+};
+
+extern char args_doc[];
+extern struct argp_option options[];
+extern struct argp argp;
+
+error_t parse_opt (int key, char *arg, struct argp_state *state)
```

Comment:
Just to be clear, are you referring to the entire args.h file? If so, I'll remove this header and paste the args code back into the files that need it.